<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageEditorContainer</class>
 <widget class="QWidget" name="PageEditorContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>341</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <item>
        <widget class="QToolButton" name="addPageButton">
         <property name="toolTip">
          <string>Add new page</string>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="page_editor.qrc">
           <normaloff>:/images/document-new.png</normaloff>:/images/document-new.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="deletePageButton">
         <property name="toolTip">
          <string>Remove current page</string>
         </property>
         <property name="text">
          <string>Del</string>
         </property>
         <property name="icon">
          <iconset resource="page_editor.qrc">
           <normaloff>:/images/document-close.png</normaloff>:/images/document-close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="clonePageButton">
         <property name="toolTip">
          <string>Make a copy of current page</string>
         </property>
         <property name="text">
          <string>Copy</string>
         </property>
         <property name="icon">
          <iconset resource="page_editor.qrc">
           <normaloff>:/images/document-multiple.png</normaloff>:/images/document-multiple.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pageMoveFront">
         <property name="toolTip">
          <string>Page move front</string>
         </property>
         <property name="text">
          <string>front</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../addons_pro/plugins/core_plugins/core/page/page_ext.qrc">
           <normaloff>:/images/move_left.png</normaloff>:/images/move_left.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="pageMoveBack">
         <property name="toolTip">
          <string>Page move back</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
         <property name="icon">
          <iconset resource="../../../../addons_pro/plugins/core_plugins/core/page/page_ext.qrc">
           <normaloff>:/images/move_right.png</normaloff>:/images/move_right.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ItemsToolWidget" name="tools" native="true"/>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolBar" name="actions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="ItemTools" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="FontEditor" name="fontEditor" native="true"/>
        </item>
        <item>
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="AlignmentEditor" name="alignmentEditor" native="true"/>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="FrameEditor" name="frameEditor" native="true"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>570</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="pageLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="FancyTabWidget" name="pageTabs" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QWidget" name="widget" native="true">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>150</y>
            <width>211</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="ObjectInspectorLayout"/>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="PropertyEditorLayout"/>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FancyTabWidget</class>
   <extends>QWidget</extends>
   <header location="global">fancytabs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ItemsToolWidget</class>
   <extends>QWidget</extends>
   <header>itemstoolwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FontEditor</class>
   <extends>QWidget</extends>
   <header>fonteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AlignmentEditor</class>
   <extends>QWidget</extends>
   <header>alignmenteditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameEditor</class>
   <extends>QWidget</extends>
   <header>frameeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../addons_pro/plugins/core_plugins/core/page/page_ext.qrc"/>
  <include location="page_editor.qrc"/>
 </resources>
 <connections/>
</ui>
